src/DynamoDbModel.php

```diff
@@ -195,10 +218,10 @@ public static function all($columns = [], $limit = -1)
         return $model->getAll($columns, $limit);
     }
 
-    public static function first($columns = [])
+    public function first($columns = [])
```

Comment:
that is why i had this https://github.com/baopham/laravel-dynamodb/blob/6e440dbb961302572e6dfa2013973b77a58e544d/src/DynamoDbModel.php#L75 - I did not look at the changes carefully in the previous PR.